README.md

```diff
@@ -184,6 +184,24 @@ For an example of what this might look like in practice, see below.
 
 ![alt text](exampleDesign.png)
 
+<!-- USER GLOBAL CONTEXT -->
```

Comment:
Helpful documentation, very easy to follow, but in the READme we have the space to be descriptive. It may be beneficial to add a line or two describing what this change is and why it's necessary.

Reply:
I getchya kinda like a mini introduction? Or do you mean literally putting some extra lines in to separate sections lol, I might be over complicating it.

Reply:
Not over complicating it, just a few sentences explaining the benefits of incorporating global context and how this PR is meeting this end.

Reply:
Noice! Ok I will make those changes and push up here in a beat!

Reply:
Right on, thanks!